var/spack/repos/builtin/packages/cantera/package.py

```diff
@@ -0,0 +1,197 @@
+
+    # Required dependencies
+    depends_on('scons')
+
+    # Recommended dependencies
+    depends_on('blas',      when='+lapack')
+    depends_on('lapack',    when='+lapack')
+    depends_on('boost',     when='+threadsafe')
+    depends_on('sundials',  when='+sundials')  # must be compiled with -fPIC
+
+    # Python module dependencies
+    extends('python', when='+python')
+    depends_on('py-numpy',  when='+python')
+    depends_on('py-scipy',  when='+python')
+    depends_on('py-cython', when='+python')
+    depends_on('py-3to2',   when='+python')
+    # TODO: these "when" specs don't actually work
+    # depends_on('py-unittest2',     when='+python^python@2.6')
+    # depends_on('py-unittest2py3k', when='+python^python@3.1')
+
+    # Matlab toolbox dependencies
+    # TODO: add Matlab package
+    # TODO: allow packages to extend multiple other packages
+    # extends('matlab',   when='+matlab')
+
+    def install(self, spec, prefix):
+        # Required options
+        options = [
+            'prefix={0}'.format(prefix),
+            'CC={0}'.format(os.environ['CC']),
+            'CXX={0}'.format(os.environ['CXX']),
+            'F77={0}'.format(os.environ['F77']),
+            'FORTRAN={0}'.format(os.environ['FC']),
+            'cc_flags=-fPIC',
+            # Allow Spack environment variables to propagate through to SCons
+            'env_vars=all'
+        ]
+
+        # BLAS/LAPACK support
+        if '+lapack' in spec:
+            options.extend([
+                'blas_lapack_libs=lapack,blas',
+                'blas_lapack_dir={0}'.format(spec['lapack'].prefix.lib)
+            ])
+
+        # Threadsafe build, requires Boost
+        if '+threadsafe' in spec:
+            options.extend([
+                'build_thread_safe=yes',
+                'boost_inc_dir={0}'.format(spec['boost'].prefix.include),
+                'boost_lib_dir={0}'.format(spec['boost'].prefix.lib),
+                'boost_thread_lib=boost_thread-mt,boost_system-mt'
+            ])
+        else:
+            options.append('build_thread_safe=no')
+
+        # Sundials support
+        if '+sundials' in spec:
+            options.extend([
+                'use_sundials=y',
+                'sundials_include={0}'.format(spec['sundials'].prefix.include),
+                'sundials_libdir={0}'.format(spec['sundials'].prefix.lib),
+                'sundials_license={0}'.format(
+                    join_path(spec['sundials'].prefix, 'LICENSE'))
+            ])
+        else:
+            options.append('use_sundials=n')
+
+        # Python module
+        if '+python' in spec:
+            options.extend([
+                'python_package=full',
+                'python_cmd={0}'.format(
+                    join_path(spec['python'].prefix.bin, 'python')),
+                'python_array_home={0}'.format(spec['py-numpy'].prefix)
+            ])
+            if spec['python'].satisfies('@3'):
```

Comment:
Actually, I'll keep it for now. SCons is a build dependency, not a runtime dependency. If you remove the SCons dependency, you can use an external SCons and build Cantera with Python 3.